core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -879,6 +879,8 @@ enum class UniqueType(
     ForEveryAmountCountable("for every [amount] [countable]", UniqueTarget.MetaModifier),
     ModifiedByGameSpeed("(modified by game speed)", UniqueTarget.MetaModifier,
         docDescription = "Can only be applied to certain uniques, see details of each unique for specifics"),
+    AIPriorityModifier("[relativeAmount]% priority", UniqueTarget.MetaModifier,
```

Comment:
A. "[Amount] weight for AI decisions"
B. Target type Event, not applicable to other objects yet
C. Should be invisible to users
D. doc description is lacking - as a modder, I wouldn't know what numbers to put here or what they mean

Reply:
Your suggested syntax implies that the AI priorities should be done in overwriting, not stackable manner. Why is that?